frontend/src/components/navigation/MenuItems.js

```diff
@@ -0,0 +1,34 @@
+export const MenuItems = [
```

Comment:
I like this setup! I think you could've just put the links directly into the different files, but I have no problem with increase modularity 😁 

Important note: You should either move this within the `Header.tsx` file, or keep this file but make it a TypeScript React file, where the `MenuItems` object has a type. You can view other examples in the database for how to define a type -- this one could be a `MenuItem[]`, for example, where `MenuItem` has the `title` and `url` attributes (class name no longer applicable as we are using styled components).